airbyte-oauth/src/main/java/io/airbyte/oauth/flows/FacebookMarketingOAuthFlow.java

```diff
@@ -0,0 +1,74 @@
+package io.airbyte.oauth.flows;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.config.persistence.ConfigRepository;
+import io.airbyte.oauth.BaseOAuthFlow;
+import java.io.IOException;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * Following docs from https://developers.facebook.com/docs/facebook-login/manually-build-a-login-flow
+ */
+public class FacebookMarketingOAuthFlow extends BaseOAuthFlow {
+
+  private static final String CONSENT_URL = "https://www.facebook.com/v11.0/dialog/oauth";
+  private static final String ACCESS_TOKEN_URL = "https://graph.facebook.com/v11.0/oauth/access_token";
```

Comment:
I am guessing we need to look at https://github.com/airbytehq/airbyte/issues/5993 to encode the API's version properly here?